resolver/addressable_resolver.go

```diff
@@ -158,9 +158,18 @@ func (r *URIResolver) URIFromObjectReference(ctx context.Context, ref *corev1.Ob
 		return nil, apierrs.NewNotFound(gvr.GroupResource(), ref.Name)
 	}
 
+	lister, err := r.listerFactory(gvr)
+	if err != nil {
+		return nil, apierrs.NewNotFound(gvr.GroupResource(), "Lister")
+	}
+
+	obj, err := lister.ByNamespace(ref.Namespace).Get(ref.Name)
+	if err != nil {
+		return nil, apierrs.NewNotFound(gvr.GroupResource(), ref.Name)
+	}
+
```

Comment:
I know this code was here before but is there not a distinction between an error and object `not found`

Reply:
Yeah, always a bit odd to see a dropped err value